src/detect-engine-port.c

```diff
@@ -2391,6 +2394,50 @@ static int PortTestFunctions06(void)
 }
 
 /**
+ * \test Test general functions
+ */
+static int PortTestFunctions07(void)
```

Comment:
Can you rewrite this to use the new FAIL_IF/PASS macro's? See https://github.com/inliniac/suricata/blob/master/src/detect-tls-cert-validity.c#L643 for a simple example.